src/main/java/com/mesosphere/velocity/marathon/impl/MarathonBuilderImpl.java

```diff
@@ -327,16 +329,16 @@ private void setLabels() {
     }
 
 
-    private JSONObject setEnv() {
-        if (!json.has("env"))
-            json.element("env", new JSONObject());
+    private void setEnv() {
+        if (config.getEnv() != null && config.getEnv().size() > 0) {
```

Comment:
not a big deal... but if commons-util or commons-collection is in the dependency chain already... I would rather see `CollectionUtils.isNotEmpty(config)`